Documentation/policy/language.rst

```diff
@@ -81,6 +87,57 @@ all ingress traffic to an endpoint may be done as follows:
 
         .. literalinclude:: ../../examples/policies/l3/ingress-allow-all/ingress-allow-all.json
 
+Egress
+~~~~~~
+
+An endpoint is allowed to send traffic to another endpoint if at least one
+egress rule exists which selects the destination endpoint with the 
+`EndpointSelector` in the ``endpointSelector`` field. To restrict traffic upon
+egress to the selected endpoint, the rule selects the destination endpoint with
+the `EndpointSelector` in the ``toEndpoints`` field.
+
+Simple Egress Allow
+~~~~~~~~~~~~~~~~~~~~
+
+The following example illustrates how to use a simple egress rule to allow
+communication to endpoints with the label ``role=backend`` from endpoints with
+the label ``role=frontend``.
+
+.. only:: html
+
+   .. tabs::
+     .. group-tab:: k8s YAML
+
+        .. literalinclude:: ../../examples/policies/l3/simple/l3_egress.yaml
+     .. group-tab:: JSON
+
+        .. literalinclude:: ../../examples/policies/l3/simple/l3_egress.json
+
+.. only:: epub or latex
+
+        .. literalinclude:: ../../examples/policies/l3/simple/l3_egress.json
+
+
+Egress Allow All
```

Comment:
Would it be worthwhile to point out here that you can allow all egress but an endpoint with ingress rules would not allow anything to connect to it. This might be blatantly obvious (it is to us) but I don't have a sense for how much we want our doc sections to be independently complete.
A similar note might be useful in the examples above. In particular, noting that the egress allow policy allows `frontend` pod outbound connections to `backend` pods but the `backend` pod ingress policy must allow connections for stuff to work (the note can be much shorter, I'm just explaining).

Reply:
I provided a general note for both ingress and egress, which I think is enough to point users in the right direction. Let me know what you think.